examples/examples/opentelemetry.rs

```diff
@@ -33,29 +33,28 @@ fn init_tracer() -> io::Result<()> {
             ..Default::default()
         })
         .build();
-    global::set_provider(provider);
+    let tracer = provider.get_tracer("tracing");
+
+    let opentelemetry = tracing_opentelemetry::layer().with_tracer(tracer);
+    let subscriber = Registry::default().with(opentelemetry);
+    tracing::subscriber::set_global_default(subscriber)?;
 
     Ok(())
```

Comment:
nit, take it or leave it: maybe consider
```suggestion
    tracing_subscriber::registry()
        .with(opentelemetry)
        .try_init()
```
(although this will require importing `tracing_subscriber::prelude::*`)